boot/bootdata/packages/reactos.dff.in

```diff
@@ -147,6 +147,8 @@ Signature = "$Windows NT$"
 "modules/optional/vmx_mode.dll"                       2  optional
 "modules/optional/vmx_svga.inf"                       6  optional
 "modules/optional/vmx_svga.sys"                       3  optional
+"modules/optional/vmwogl32.dll"                       2  optional
```

Comment:
An x64 version of the VMware video driver also has an OpenGL ICD component for x64 OpenGL applications.
```suggestion
"modules/optional/vmwogl32.dll"                       2  optional
"modules/optional/vmwogl64.dll"                       2  optional
```

Reply:
It would, here's VMware OpenGL running on x64:
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/2d63719e-a918-4c99-b9f9-d0e9043d2ac8" />